workflow/commit-messages.md

```diff
@@ -0,0 +1,34 @@
+# Commit Guidelines
+
+As a general guideline you can follow the idea that *a single commit should enclose a single problem/feature that has been fixed/added*.
+
+This way of organizing your changes to the codebase makes is easy for reviewers and people who try to understand the history of a project to determine what has been happening in the project.
+
+Here a both a good and a bad example of of commit message.
+
+### BAD
+```
+fixes bug
+```
+### GOOD
+```
+Fix bug where database gets deleted on POST request
+```
+
+## Breaking up large tickets
+
+If your feature/ticket includes mutliple things that you have to complete in order to submit a pull request, these should all be seperate commit messages, according to the general guideline above. This will make it easy for the reviewer to understand what exactly has happened.
+
+For adding a new page in an example Project this could look like the following
+```
+Add the Person model
+Configure the Views to use the new Person Model
+Use the Data from the Views in the index template
+```
+
+Further information about how to write good commit messages can be found [here](https://chris.beams.io/posts/git-commit/)
```

Comment:
When linking, it would be nice to include more descriptive link text, rather than "here" or "click here". Maybe this sentence could be restructured to accommodate that as something like:

> Chris Beams has written [a good article on how to write commit messages](https://chris.beams.io/posts/git-commit/).

This also achieves giving credit to the author, which is supposed to be one of our values. (And maybe this linking point should be added to a new document about writing for the web at some point).